pkgs/top-level/all-packages.nix

```diff
@@ -33454,6 +33454,8 @@ with pkgs;
 
   linuxsampler = callPackage ../applications/audio/linuxsampler { };
 
+  listen1 = callPackage ../applications/audio/listen1 { };
```

Comment:
Please use the `pkgs/by-name` hierarchy, read `pkgs/by-name/README.md` for instructions.